draft-ietf-rats-architecture.md

```diff
@@ -360,6 +363,42 @@ are signed.  For example, values might have been generated at boot, and then
 used in claims as long as the signer can guarantee that they cannot have changed
 since boot.
 
+The term freshness is applied to both Evidence and Attestation Results in
+this document. Unfortunately, the term freshness is also a bit overloaded and
+in order to illustrate the application of the term in this document better,
+two characteristics that can be part of freshness are elaborated on below:
+recentness and uniqueness:
+
+Recentness:
+
+: a quality of Evidence that is based on the time that has passed between
+creation of the evidence by an Attester (typically by "sealing" a Claims set
+via signing) and the reception of that Evidence by a Verifier or (relaying
+party).
+
+: Evidence can be not recent enough, if the Evidence created from collected
+Claims is too old. A nonce supplied during a Challenge/Response handshake and
```

Comment:
I'm slightly confused: nonce is about uniqueness, not recentness.

Reply:
pls, see below

Reply:
Sorry, I can't see anything below, even after refreshing the page.

Reply:
Sry. "Below" was meant to refer to a different comment of yours on lines 376-379 (I am not sure how to ref that comment here).

Reply:
@thomas-fossati the nonce is about freshness.  See discussion of examples 2 and 4 in sections 15.2 and 15.4 in the editors copy.

Reply:
@dthaler I certainly agree nonce is about freshness.  What I am arguing here (and in the following, long-winded discussion) is that a nonce creates freshness by "uniqueness" rather than "recentness".